Vector/ViewController/MediaPickerViewController.m

```diff
@@ -1506,6 +1498,36 @@ - (void)moviePlayerThumbnailImageRequestDidFinishNotification:(NSNotification *)
     {
         validationView.image = [[notification userInfo] objectForKey:MPMoviePlayerThumbnailImageKey];
         [validationView bringSubviewToFront:videoPlayerControl];
+
+        // Now, there is a thumbnail, show the video control
+        videoPlayerControl = [UIButton buttonWithType:UIButtonTypeCustom];
+        [videoPlayerControl addTarget:self action:@selector(controlVideoPlayer) forControlEvents:UIControlEventTouchUpInside];
+        videoPlayerControl.frame = CGRectMake(0, 0, 44, 44);
+        [videoPlayerControl setImage:[UIImage imageNamed:@"camera_play"] forState:UIControlStateNormal];
+        [videoPlayerControl setImage:[UIImage imageNamed:@"camera_play"] forState:UIControlStateHighlighted];
+        [validationView addSubview:videoPlayerControl];
+        videoPlayerControl.center = validationView.imageView.center;
+
+        videoPlayerControl.translatesAutoresizingMaskIntoConstraints = NO;
+        NSLayoutConstraint *centerXConstraint = [NSLayoutConstraint constraintWithItem:videoPlayerControl
+                                                                             attribute:NSLayoutAttributeCenterX
+                                                                             relatedBy:NSLayoutRelationEqual
+                                                                                toItem:validationView.imageView
+                                                                             attribute:NSLayoutAttributeCenterX
+                                                                            multiplier:1
+                                                                              constant:0.0f];
+
+        NSLayoutConstraint *centerYConstraint = [NSLayoutConstraint constraintWithItem:videoPlayerControl
+                                                                             attribute:NSLayoutAttributeCenterY
+                                                                             relatedBy:NSLayoutRelationEqual
+                                                                                toItem:validationView.imageView
+                                                                             attribute:NSLayoutAttributeCenterY
+                                                                            multiplier:1
+                                                                              constant:0.0f];
+
+        [NSLayoutConstraint activateConstraints:@[centerXConstraint, centerYConstraint]];
+
+        [videoPlayerControl layoutIfNeeded];
```

Comment:
Do this 'layoutIfNeeded' is required? If yes it should be applies on validationView